src/ica_api.c

```diff
@@ -1273,7 +1276,9 @@ unsigned int ica_rsa_crt(ica_adapter_handle_t adapter_handle,
 	rb.outputdata = output_data;
 	rb.outputdatalength = rsa_key->key_length;
 
-	ica_rsa_crt_key_check(rsa_key);
+	rc = ica_rsa_crt_key_check(rsa_key);
+	if (rc > 1)
+		return rc;
```

Comment:
in line 1310 you cleans rb before returning. Should we also cleanse it here in that error case ?

Reply:
The sensitive information is copied into rb below this return, so I think we can leave it as it is.

Reply:
Actually `rb` only contains pointers, so it does not contain any sensitive information anyway.